README.md

```diff
@@ -181,6 +181,91 @@ defmodule YourApp.Router do
     ]
 ```
 
+## Custom data
+By default, `BoomNotifier` will **not** include any custom data from your 
+requests.
+
+However, there are different strategies to decide which information do
+you want to include in the notifications using the `:custom_data` option 
+with one of the following values: `:assigns`, `:logger` or both.
+
+The included information will show up in your notification, in a new section 
+titled "Data".
+
+### Assigns
+This option will include the data that is in the [connection](https://hexdocs.pm/plug/Plug.Conn.html)
+`assigns` field.
+
+You can also specify the fields you want to retrieve from `conn.assigns`.
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    custom_data: :assigns,
+    notifiers: [
+      # ...
+    ]
+```
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    custom_data: [assigns: [fields: [:current_user, :session_data]]],
+    notifiers: [
+      # ...
+    ]
+```
+
+Example of adding custom data to the connection:
+
+```elixir
+conn
+|> assign(:name, "John")
+```
+
+### Logger
+This option will include the data that is in the [Logger](https://hexdocs.pm/logger/Logger.html)
+`metadata` field.
+
+You can also specify the fields you want to retrieve from `Logger.metadata()`.
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    custom_data: :logger,
+    notifiers: [
+      # ...
+    ]
+```
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    custom_data: [logger: [fields: [:request_id, :current_user]]],
+    notifiers: [
+      # ...
+    ]
+```
+
+Example of adding custom data to the logger:
+
+```elixir
+Logger.metadata(name: "John")
+```
+
+### Using both
+
+You can do any combination of the above settings to include data
+from both sources.
+
```

Comment:
I agree it would be nice to have an example with the two notifiers. However, I think both are going to be returned in their own "namespace" so it shouldn't be any conflict.


```elixir
assert %{assigns: %{name: "Davis"}, logger: %{age: 17}} = data
```

https://github.com/wyeworks/boom/pull/37/files#diff-27637604d9b52ac66b458efdee984aa2766bc282f1062b6e5ff6497134593f5bR232-R238

Reply:
Yes, I agree that it would be clear to add an example using both and a note saying that the field names are independent of each source. As @ iaguirre88 said, we are returning them in their own "namespace".

I will add a test for that as well.